sonaranalyzer-dotnet/its/expected/akka.net/Akka.TestKit-{0D3CBAD0-BBDB-43E5-AFC4-ED1D3ECDC224}-S4039.json

```diff
@@ -0,0 +1,329 @@
+{
+"issues":  [
+{
+"id":  "S4039",
+"message":  "Make 'TestActorRefBase' sealed, change to a non explicit declaration or provide a new method exposing the functionality of 'ISurrogated.ToSurrogate'.",
```

Comment:
Quite a few reports on test files. Do we actually care if the method is not accessible in test? I'd say it doesn't matter and we should ignore them.
WDYT?